src/app/reskin/components/header.tsx

```diff
@@ -0,0 +1,21 @@
+"use client";
+
+import { themeAtom } from "@/util/atoms";
+import { useAtom } from "jotai";
+
+export default function Header() {
+  const [theme, setTheme] = useAtom(themeAtom);
+  return (
+    <header>
+      <div className="mx-auto max-w-7xl px-4 py-6 sm:px-6 lg:px-8 flex justify-between">
+        <h1 className="text-3xl font-bold tracking-tight">SBTC Bridge</h1>
```

Comment:
not sure if this comes from some mock, but usually we use `sBTC`

Reply:
this is still WIP actual header is next